packages/app/src/modules/current.js

```diff
@@ -69,11 +69,13 @@ export const getOptionsFromUi = ui => {
     })
 
     // cast option values to Number for some options
-    ;['sortOrder', 'topLimit'].forEach(option => {
-        if (Object.prototype.hasOwnProperty.call(optionsFromUi, option)) {
-            optionsFromUi[option] = Number(optionsFromUi[option])
+    ;['baseLineValue', 'targetLineValue', 'sortOrder', 'topLimit'].forEach(
+        option => {
+            if (Object.prototype.hasOwnProperty.call(optionsFromUi, option)) {
+                optionsFromUi[option] = Number(optionsFromUi[option])
```

Comment:
Could this ever return `NaN`?

Reply:
It shouldn't as the UI uses numeric input fields.